apps/callscreen/js/calls_handler.js

```diff
@@ -369,7 +369,7 @@ var CallsHandler = (function callsHandler() {
    *
    * @returns {Integer} The number of calls currently present.
    */
-  function openLines() {
+  function numOpenLines() {
```

Comment:
This function is a great candidate for unit testing. Writing tests for it specifically would fall more in line with the spirit of unit testing than the way we're currently doing them. But this is more just thinking aloud, and I won't hold this patch back for this issue.